docs/api-reference/mapbox/mapbox-overlay.md

```diff
@@ -99,8 +99,8 @@ export default function App() {
   return (
     <Map
       initialViewState={{
-        latitude: 37.78,
-        longitude: -122.45,
+        latitude: 40,
+        longitude: -74.5,
         zoom: 12
       }}
       mapStyle="mapbox://styles/mapbox/light-v9"
```

Comment:
I'd like to change this to maplibre so that it would work copy-paste or at least link to https://visgl.github.io/react-map-gl/docs/get-started/get-started#using-with-a-mapbox-gl-fork for a quick change.